packages/flutter_tools/templates/create/android.tmpl/buildSrc/src/main/groovy/FlutterPlugin.groovy

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
We presumably have to be able to hook into the build at some point, to build the Dart code and zip up the assets, for example. Can we provide the code at that point?